src/main/java/uk/gov/cslearning/record/service/scheduler/LearningJob.java

```diff
@@ -64,7 +72,89 @@ public LearningJob(UserRecordService userRecordService, IdentityService identity
         this.notificationRepository = notificationRepository;
     }
 
-    @Transactional
+    private void CheckAndNotifyLineManager(CivilServant civilServant, Identity identity,Course course, LocalDateTime completedDate) throws NotificationClientException {
+
+        if (civilServant.getLineManagerEmail() == null) {
+            LOGGER.error("User has no line manager!");
+        } else {
+            Boolean sendMail = false;
```

Comment:
I think you can use the lower-case, primitive `boolean` here. It will have lower memory overheads

Reply:
okay sure